src/buildx.ts

```diff
@@ -111,3 +110,29 @@ async function download(version: string): Promise<string> {
 
   return await tc.cacheFile(downloadPath, targetFile, 'buildx', version);
 }
+
+async function filename(version: string): Promise<string> {
+  let arch: string;
+  switch (context.osArch) {
+    case 'x64': {
+      arch = 'amd64';
+      break;
+    }
+    case 'ppc64': {
+      arch = 'ppc64le';
+      break;
+    }
+    case 'arm': {
+      const arm_version = (process.config.variables as any).arm_version;
+      arch = arm_version ? 'arm-v' + arm_version : 'arm';
```

Comment:
I think we should change upstream to `armv7` instead of `arm-v7` so these hacks are not required(they are especially tricky in Dockerfiles). I'll try to remember to update this when we do.